src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateTimeValue_TimeZone_Paris.txt

```diff
@@ -0,0 +1,28 @@
+﻿// (UTC+01:00) Brussels, Copenhagen, Madrid, Paris
+#SETUP: TimeZoneInfo("Romance Standard Time")
+
+>> DateTimeValue("jeudi 21 juillet 2022 19:34:03", "fr-FR")
+DateTime(2022,7,21,19,34,3,0)
+
+>> DateTimeValue("jeudi 21 juillet 2022 19:34:03", "unknown")
+#Error(Kind=BadLanguageCode)
+
+// The result of this expression depends on current TimeZoneInfo. 
```

Comment:
It would be nice to add some tests that parse data from UTC. `DateTimeValue("2022-08-23T12:00:00Z")` should result in `DateTime(2022,8,23,5,0,0,0)` in Seattle, but `DateTime(2022,8,23,14,0,0,0)` (I think) in Paris.

Reply:
That's an excellent test for which I had to update DateTimeValue. Good catch!